ui/components/multichain/app-footer/app-footer.js

```diff
@@ -151,7 +157,9 @@ export const AppFooter = () => {
                 href="#"
                 onClick={(e) => {
                   e.preventDefault();
-                  history.push(CONNECTIONS);
+                  const isFullScreen =
+                    getEnvironmentType() === ENVIRONMENT_TYPE_FULLSCREEN;
```

Comment:
I was thinking it is more clear to move this logic outside the render and clear to read. 
```
const isFullScreen = useMemo(() => getEnvironmentType() === ENVIRONMENT_TYPE_FULLSCREEN, [getEnvironmentType]);

Reply:
Due to getEnvironmentType() being a function used outside of the component, it would not be a valid dependency for useMemo